redaxo/src/addons/media_manager/lib/managed_media.php

```diff
@@ -190,6 +192,11 @@ public function setImage($src)
         $this->asImage = true;
     }
 
+    public function setImageAttribute($key, $value)
```

Comment:
setImageProperty = Eigenschaft
Spezifische Methodennamen wie setImageQuality finde ich aber immer die bessere Wahl weil die ja dann auch in der API Doku vernünftig auftauchen :)